sql-join/magic-mallrats.sql

```diff
@@ -0,0 +1,6 @@
+select "customers"."firstName", "customers"."lastName"
+from "customers"
+join "rentals" on "customers"."customerId" = "rentals"."customerId"
+join "inventory" on "rentals"."inventoryId" = "inventory"."inventoryId"
+join "films" on "inventory"."filmId" = "films"."filmId"
```

Comment:
It's nice that you are getting experience using both syntaxes, but `using` is more succinct, if the column names are the same.